docs/current/quickstart/947391-test.mdx

```diff
@@ -83,14 +83,14 @@ The `from()`, `withDirectory()`, `withWorkdir()` and `withExec()` methods all re
 </TabItem>
 <TabItem value="Python">
 
-<Embed id="2XgPeWiQsRX" />
+<Embed id="yZj9P2N03-D" />
 
 This code listing does the following:
 
 - It creates a Dagger client with `with dagger.Connection()` as before.
 - It uses the client's `container().from_()` method to initialize a new container from a base image - again, the `node:16-slim` image. This base image is the Node.js version to use for testing. The `from_()` method returns a new `Container` object with the result.
 - It uses the `Container.with_directory()` method to mount the source code directory on the host at the `/src` mount point in the container, and the `Container.with_workdir()` method to set the working directory to that mount point.
-  - Notice that the `Container.with_directory()` accepts additional options to exclude (or include) specific files from the mount. In this case, it excludes the `node_modules` (locally-installed dependencies) and `ci` (pipeline code) directories.
+  - Notice that the `Container.WithDirectory()` accepts additional options to exclude (or include) specific files from the mount. In this case, it excludes the `node_modules` (locally-installed dependencies), `ci` (pipeline code) and `build` (build artifacts) directories.
```

Comment:
```suggestion
  - Notice that the `Container.with_directory()` accepts additional options to exclude (or include) specific files from the mount. In this case, it excludes the `node_modules` (locally-installed dependencies), `ci` (pipeline code) and `build` (build artifacts) directories.
```

Reply:
Thank you! https://github.com/dagger/dagger/pull/6142